db.go

```diff
@@ -649,9 +650,10 @@ func (db *DB) close() error {
 	// Clear ops.
 	db.ops.writeAt = nil
 
+	var errs []string
```

Comment:
```suggestion
	var errs []error
```

I suggest to only return the first error if present. It is consistent with the existing behavior. We will add log in separate PR to record all errors.

Reply:
Please resolve this comment: change the type to `[]error`. It doesn't make sense to convert the errors to strings, and finally convert them back to errors.